community/benchmarks/transact_db_backends.md

```diff
@@ -0,0 +1,350 @@
+# Benchmark of Hyperledger Transact 0.3.7 LMDB and experimental SQLite backends
+<!--
+  Copyright 2018-2020 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+## Summary
+
+Hyperledger Transact provides a number of core features used by Splinter, such
+as transaction execution and state storage.  As these features play an important
+role in the operation of a splinter network, it is important to measure the
+performance of these subsystems to determine where optimizations can be made.
+
+The benchmarks included here cover the performance of one of the lowest-level
+areas: Transact's database abstraction, and two of its back-end implementations,
+LMDB and SQLite (currently an experimental feature). These benchmarks have been
+run on several file systems: Elastic Block Store (EBS), Elastic File System
+(EFS), and APFS (Apple File System).
+
+EFS has a noticeable performance impact, with a 18x slowdown for LMDB and a 99x
+slowdown for SQLite. These results strongly imply that with the current
+implementations, LMDB would be the preferred choice for a back-end implemented
+over a file on EFS.
+
+## Environment
+
+The environments where these benchmarks have been run is in AWS as well as on
+the developer's local system.  In AWS, two different file system types are used:
+Elastic Block Store (EBS) and Elastic File System (EFS).
+
+**t2.medium (edited)**
+
+* 2 vCPU
+
+  "T2 instances are backed by the latest Intel Xeon processors with clock speeds
+  up to 3.3 GHz during burst periods.”
+* 4GB RAM
+* OS: ubuntu-bionic-18.04-amd64-server-20201026
+* EBS (for a single node configuration) and EFS
+
+**Local Development Environment**
+
+* MacBook Pro (13-inch, 2020, Four Thunderbolt 3 ports)
+* Processor: 2.3 GHz Quad-Core Intel Core i7
+* Memory: 32 GB 3733 MHz LPDDR4X
+* OS: macOS Catalina 10.15.7
+
+## Setup
+
+The tests have been compiled in release mode, and include the experimental
+feature "sqlitedb" enabled:
+
+```
+cargo build --manifest-path libtransact/Cargo.toml \
+    --tests \
+    --release \
+    --features sqlite-db
+```
+
+In these benchmark runs, the integration tests compiled to a binary
+mod-5b692c58f25d0ea4 on AWS and . This includes all of the
```

Comment:
"on AWS and ." Looks like something's missing here

Reply:
removed the "and"